nextflow_schema.json

```diff
@@ -347,33 +347,32 @@
                     "help_text": "Activate this flag to generate genome plots (might be memory-intensive).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--skip-plot`"
                 }
             },
-            "fa_icon": "fas fa-file-signature",
-            "help_text": "BAKTA is a tool developed to annotate bacterial genomes and plasmids from both isolates and MAGs. \n\nDocumentation: [https://github.com/oschwengers/bakta](https://github.com/oschwengers/bakta)"
+            "fa_icon": "fas fa-file-signature"
         },
         "annotation_prokka": {
             "title": "Annotation: Prokka",
             "type": "object",
-            "description": "These parameters influence the annotation algorithm used by Prokka.",
+            "description": "Prokka annotates genomic sequences belonging to bacterial, archaeal and viral genomes. More info: [Prokka documentation](https://github.com/tseemann/prokka)",
```

Comment:
Markdown links don't work in the descriptions, replace with a raw link

Reply:
![image](https://github.com/nf-core/funcscan/assets/17950287/b9011bb8-07b1-4168-bdd3-95a01f6708d2)

Applies to all sections!